aiokafka/admin/client.py

```diff
@@ -27,6 +27,7 @@
 from aiokafka.protocol.commit import GroupCoordinatorRequest, OffsetFetchRequest
 from aiokafka.protocol.metadata import MetadataRequest
 from aiokafka.structs import OffsetAndMetadata, TopicPartition
+from aiokafka.admin import NewPartitions
```

Comment:
It would be better to avoid importing from parent package
```suggestion
from .new_partitions import NewPartitions
```

Reply:
@ods Already fixed and sort imports